docker-compose.yml

```diff
@@ -115,7 +115,7 @@ services:
       - "8101:8080"
   driver-did-trustbloc:
     container_name: driver-did-trustbloc
-    image: trustbloc/driver-did-trustbloc:0.1.4
+    image: ghcr.io/trustbloc/driver-did-trustbloc:0.1.5
```

Comment:
Pls host the image on dockerhub. We can't support other image repos.

Reply:
ghcr.io is the GitHub Container Registry. It allows anonymous access.

I'm not understanding your comment.

Reply:
Yes, the access to the registry might work but our CI script expects images in the dockerhub format `<org>/<imagename>:<tag>`. It is also stated in the [contribution guide](https://github.com/decentralized-identity/universal-resolver/blob/main/docs/driver-development.md#how-to-contribute-a-driver) that images must be hosted on dockerhub.

With the recent changes in dockerhub's pull-limits it might be worth reconcidering this restriction but as we haven't had any issues so far, we probably won't touch this topic for quiet some time.

Sorry for the initial unclear response.

Reply:
https://github.com/decentralized-identity/universal-resolver/issues/167

Reply:
@BernhardFuchs done